frontend/app/routes/_protected+/_layout.tsx

```diff
@@ -14,16 +14,16 @@ export function ErrorBoundary() {
     switch (error.status) {
       // TODO :: GjB :: handle other status codes
       default:
-        return <ServerError error={error} />;
+        return <ServerError error={error} showAccountMenu={true} />;
     }
   }
 
-  return <ServerError error={error} />;
+  return <ServerError error={error} showAccountMenu={true} />;
 }
 
 export default function Layout() {
   return (
-    <ApplicationLayout>
+    <ApplicationLayout showAccountMenu={true}>
```

Comment:
[Props Default to “True”](https://legacy.reactjs.org/docs/jsx-in-depth.html#props-default-to-true)

Reply:
Fixed